x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -82,6 +82,8 @@ filebeat.inputs:
     client.id: 12345678901234567890abcdef
     client.secret: abcdef12345678901234567890
     token_url: http://localhost/oauth2/token
+    user: abc.xyz@mail.com
```

Comment:
Mail.com is a product https://www.mail.com/int/ :) Let's not suggest anything. How about user@domain.tld?

Reply:
sure :)  its updated.